### ISSUE TITLE
Moving to LLVM 9.0.0

### ISSUE THREAD

**Issue by lifeasageek** [SVF-tools/SVF#142]
Hello,

I've prepared the patch for LLVM 9.0.0 (this seems to build the Linux kernel):
https://github.com/lifeasageek/SVF/commit/b7c0922ade78da2e13340e25fb5a405ba80b5033

I haven't fully worked through if this is right, so please let me know if something's wrong.

**Comment by yuleisui** [comment]
Thanks! We have a look and test once after the official release of LLVM-9.0. If you have any updates, please feel free to post.

**Comment by Da5hes** [comment]
@yuleisui LLVM 9.0.0 has just been released. would you mind looking at it once more?

**Comment by yuleisui** [comment]
Sure, will try to update to llvm 9 these two days. It seems that the terminaterInst has gone?

**Comment by Da5hes** [comment]
yes it was removed in 8.0.0, see https://reviews.llvm.org/D47467

**Comment by yuleisui** [comment]
The build breaks in my macOS when I changed "add_llvm_loadable_module" to "add_llvm_library" in CMakeLists.txt. Any idea about this?

**Comment by yuleisui** [comment]
Now SVF has been upgraded to LLVM-9.0.0, thanks
